modules/compliance-scheduling-pods-with-resource-requests.adoc

```diff
@@ -51,6 +59,11 @@ spec:
       limits:
         memory: "128Mi"
         cpu: "500m"
+    # TODO: Is this correct, added to each container?
```

Comment:
@s-urbaniak @stlaz If there are multiple containers listed, does the securityContext need to be added to each one, like this?

Reply:
yes, unfortunately